ncj/imagemagick/resize-images/job.template.json

```diff
@@ -0,0 +1,97 @@
+{
+   "templateMetadata":{
+       "description":"Sample application template for working with ImageMagick."
+   },
+   "parameters":{
```

Comment:
can you add a poolId parameter that set the pool info. BatchLabs override it but is needed to be able to use this template from the command line

Reply:
ok cool. i copied the 3ds max one as a base so it's likely that that one doesn't have it either.